portfolio/src/main/java/com/google/sps/servlets/ChartServlet.java

```diff
@@ -14,23 +14,29 @@
 
 package com.google.sps.servlets;
 
-import com.google.sps.Events;
 import com.google.gson.Gson;
+import java.io.BufferedReader;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
 import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-@WebServlet("/get-events")
-public class GetEventsServlet extends HttpServlet {
+/* Servelt to read chart data from CSV and return it in doGet */
```

Comment:
nit: Servlet

Reply:
I'll rename it and reflect the update in a new PR.